keps/draft-20180426-security-conformance.md

```diff
@@ -0,0 +1,136 @@
+---
+kep-number: draft-20180426
+title: Security Conformance
+authors:
+  - "@easeway"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-testing
+  - sig-api-machinery
+  - sig-network
+  - sig-node
+  - sig-cluster-lifecycle
+reviewers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+  - "@davidopp"
+approvers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+editor: "@easeway"
+creation-date: 2018-04-26
+status: provisional
+---
+
+# Security Conformance
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [List of Criteria](#list-of-criteria)
+* [Graduation Criteria](#graduation-criteria)
+
+## Summary
+
+Security Conformance defines citeria to certify a Kubernetes cluster to be minimally secure.
+
+## Motivation
+
+Configuring a Kubernetes cluster is complicated,
+and it's very easy to mis-configure a cluster to be insecure.
+The current Kubernetes conformance test suite barely covers security aspects.
+
+### Goals
+
+- Define the initial list (to evolve later) of criteria for minimal security requirements
+  - Define the role of the actor (unprivileged user, namespace admin, cluster admin, etc.)
+  - Define pre-requisites to run test for verifying the criteria
+  - Define test behaviors
+  - Define test expectations
+
+### Non-Goals
+
+- A full list of criteria at the beginning
+  - The list of criteria is to be curated by the community
+  - This proposal starts with minimal criteria to be evolved quickly
+- Mechanisms to implement the test
+- Mechanisms to setup environment for the tests
+- Host setup and control plane configurations (covered by CIS benchmark)
+- Non-security related
+
+## Proposal
+
+### List of Criteria
```

Comment:
Is this a complete list, or a starter list? (It seems you wanted to design the list of criteria elsewhere - where is that happenign?)

Reply:
It's a starter list.